docs/Community/sig-architecture/README.md

```diff
@@ -0,0 +1,26 @@
+Architecture Special Interest Group
+===================================
+
+SIG-Architecture is responsible for reviewing high level designs, which inform project goals and technical norms.
```

Comment:
I am not familiar with processes here. Does SIG-Attestation seek consensus from SIG-Architecture? The phrasing here suggests SIG -Architecture must get consensus from other SIGS. I am wondering if the reverse applies as well.

Reply:
great question, @radhikaj -- the way I've seen this handled in other open source communities is that a group of technical leaders (representatives, if you will) meet regularly to hash out large architectural questions for the project, agree on "a plan", which is then recorded as a design document.

Each design should include the input of which ever team(s) are ultimately going to do the work and maintain it.

So I think my answer is -- it goes both ways.